bin/nim-gdb.bash

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# Exit if anything fails
+set -e
+
+which nim > /dev/null || (echo "nim not in PATH"; exit 1)
+which gdb > /dev/null || (echo "gdb not in PATH"; exit 1)
+which readlink > /dev/null || \
+    which greadline > /dev/null || \
+    ([[ "$OSTYPE" != "darwin"* ]] && \
+        (echo "readlink not in PATH. Please install coreutils from homebrew."; exit 1)) || \
+    (echo "readlink not in PATH."; exit 1)
+
+nreadlink () {
```

Comment:
This function is needed because a variable can't be called in the below:

```bash
NIM_SYSROOT=$(dirname $(dirname $(nreadlink -e $(which nim))))
```

If `nreadlink` is instead a variable, the command will fail to run.